src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -126,16 +126,15 @@ foam.CLASS({
         scroller.value += negative ? -rows : rows;
       });
       this.container_ = parent.start().cssClass(this.myCls('table-container'));
-      this.container_.start(this.TableView, {
+      this.table = this.container_.start(this.TableView, {
```

Comment:
I've restructured this whole block to be fully fluent, using the `.call()` style.